docs/reference/migration/migrate_6_0/mappings.asciidoc

```diff
@@ -1,6 +1,17 @@
 [[breaking_60_mappings_changes]]
 === Mapping changes
 
+==== The `_all` meta field is now disabled by default
+
+On new mappings, the `_all` meta field that contains a copy of the text from
+each field is now disabled by default. The `query_string` and
+`simple_query_string` queries that previously used `_all` to search will now
+check if `_all` is enabled/disabled and switch to executing the query across all
+fields if `_all` is disabled. `_all` may be re-enabled by setting `enabled`:
```

Comment:
Thanks for your prompt response and the link to #20925. Currently in elasticsearch 5.0 I use "include_in_all: False" on everything except on text fields, so what I'll have to do with this change is indeed to create my own field with copy_to, or specify all the text fields I want in fields when I do the query.